Makefile

```diff
@@ -206,7 +206,7 @@ format:
 fmt: format
 
 lint:
-	if [ $$(wc -l < tests/utils.go) -gt 2350 ]; then echo >&2 "do not make tests/utils longer"; exit 1; fi
+	if [ $$(wc -l < tests/utils.go) -gt 2168 ]; then echo >&2 "do not make tests/utils longer"; exit 1; fi
```

Comment:
If not for this modification, the PR would have been merged. Better use a separate PR to bump this...

Reply:
You are right, next time